ToolBox/IEX/IEXDataQueueHandler.cs

```diff
@@ -68,6 +71,21 @@ public IEXDataQueueHandler() : this(true)
 
         public IEXDataQueueHandler(bool live)
         {
+            _subscriptionManager = new EventBasedDataQueueHandlerSubscriptionManager();
+            _subscriptionManager.SubscribeImpl += (s, t) =>
+            {
+                Subscribe(s);
+                return true;
+            };
+
+            _subscriptionManager.UnsubscribeImpl += (s, t) =>
+            {
+                Unsubscribe(s);
+                return true;
+            };
+
+            _subscriptionManager.GetChannelName += (t) => "quote-trade";
```

Comment:
In these cases where the channel is unique and unimportant, maybe we could define a `static string Channel.Single` or something similar? I think it will make it clearer

Reply:
done